### ISSUE TITLE
allow anonymous purchase of points?

### ISSUE THREAD

**Issue by kareila** [dreamwidth/dw-free#2435]
This came up in support but I don't see that an issue was ever opened for it:

https://www.dreamwidth.org/support/see_request?id=38263

User points out that we allow paid time to be gifted anonymously, but not points - although points can be *transferred* anonymously, as @kaberett points out.  Do we want to put the effort into adding this feature, or just let people continue to use the workaround of buying points for themselves and then transferring them anonymously?

**Comment by rahaeli** [comment]
I have a vague memory stirring in the back of my head that we did that for a reason but nothing is surfacing when I try to wade down there and kick it, so let's say yes, fix it, but low priority fix

**Comment by azurelunatic** [comment]
Was it that if there were any errors in the purchase it would be hell to track down?

**Comment by rahaeli** [comment]
Not really, or rather, not any more than existing anonymous gifts. (The only time the gifter can be sure that the gift went through without asking the giftee is if the giftee was a free account and the gifter bought paid time; they can then see the account status change on the profile. But all other gift cases don't have a confirmation, I think.) We'd still be able to look up the order if the gifter wrote in, and we can (via statushistory) figure out who the gifter was if the giftee writes in. (Disclaimers, for those reading along: we absolutely will never reveal who sent an anon gift to someone. There have been a few cases where a giftee doesn't want a gift if it came from a specific person; we still won't confirm or deny, but we will let them refuse the gift even if they aren't sure who it came from and will return the points to the gifter with an explanation if they say yes.)

**Comment by kareila** [comment]
Another request for this: https://www.dreamwidth.org/support/see_request?id=40703
